cli/commands/edit/edit.go

```diff
@@ -0,0 +1,95 @@
+package edit
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/create"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/spf13/cobra"
+)
+
+// vi is the default editor!
+const defaultEditor = "vi"
+
+func Command(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit [RESOURCE NAME] [KEY1] ... [KEYN]",
```

Comment:
`[RESOURCE NAME] [KEY1]...` is a bit confusing to me. I'd rather do `[RESOURCE] [NAME1] ... [NAMEN]` or `[RESOURCE TYPE] [NAME1] ... [NAMEN]`. WDYT?

Reply:
I agree, I was a little confused at first too. I like `[TYPE] [NAME1] ... [NAMEN]` (and add `RESOURCE` if you find that helpful). It's more consistent with our internal naming conventions.

Reply:
NAMEN :+1:

Reply:
Haha just kidding. I hate NAMEN. :laughing: 

The thing is that it's not a series of names - it's a series of key components. Not sure how best to communicate that.

Reply:
If it helps you make a decision, the typical way in Linux man pages to indicate multiple optional parameters of the same type would be `[KEY]...`. *BSD man pages would typically use `[key ...]`